go/service/gregor.go

```diff
@@ -86,14 +90,60 @@ func (g *gregorHandler) ShouldRetryOnConnect(err error) bool {
 
 func (g *gregorHandler) BroadcastMessage(ctx context.Context, m gregor1.Message) error {
 	g.G().Log.Debug("gregor handler: broadcast: %+v", m)
-	return nil
+
+	obm := m.ToOutOfBandMessage()
+	if obm == nil {
```

Comment:
This isn't necessarily an error.  If it's an in-band message, than the out-of-band message field will be nil.